src/core/annotation.js

```diff
@@ -1286,37 +1297,42 @@ class WidgetAnnotation extends Annotation {
 
     const fontInfo = await this._getFontData(evaluator, task);
     const [font, fontName] = fontInfo;
-    const fontSize = this._computeFontSize(...fontInfo, totalHeight);
     this._fontName = fontName;
 
-    let descent = font.descent;
-    if (isNaN(descent)) {
-      descent = 0;
-    }
+    const lineHeight = totalHeight;
+    const fontSize = this._computeFontSize(...fontInfo, lineHeight);
```

Comment:
The `lineHeight` doesn't seem to be used for anything else here; can you please revert these changes to reduce the overall size of the diff?